docs/projects.md

```diff
@@ -10,7 +10,7 @@
     {
         "name": "Tutorials",
         "url": "/tutorials",
-        "imageUrl": "/static/tutorials/wakanda-forever.png",
+        "imageUrl": "/static/tutorials/chase-the-pizza.png",
```

Comment:
I know this happens automatically, but do we want to remove it to keep it separate from this PR?

Reply:
My policy is to commit auto-generated files whenever they change locally (as long as they don't look suspect or dangerous).